src/client/components/ExportChart.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import Highcharts from 'highcharts'
+import { Button } from 'antd'
+require('highcharts/modules/exporting')(Highcharts)
+require('highcharts/modules/offline-exporting.js')(Highcharts)
+
+export function getExportOptions(filename) {
```

Comment:
is it okay for both the button component and this function to come from the same file? Would you say I placed this file in the correct place in the repo?